src/semgrep_agent/utils.py

```diff
@@ -42,7 +42,12 @@ def debug_echo(text: str) -> None:
     else:
         logging.info(text)
         return
-    text = "\n".join(prefix + line for line in text.splitlines())
+    try:
+        modified_text = "\n".join(prefix + line for line in text.splitlines())
+        text = modified_text
+    except Exception as e:
+        logging.info(e)
```

Comment:
Can you still log the initial error, just without the prefix?

Reply:
Ah you were suggesting that this can't be outputted at all?

Reply:
It's possible? Haven't dug into how sh is using _out and _err much yet